README.md

```diff
@@ -259,3 +259,17 @@ This will run the `detect` phase of NAB on the data files specified in the above
 JSON file. Note that scoring and normalization are not supported with this
 option. Note also that you may see warning messages regarding the lack of labels
 for other files. You can ignore these warnings.
+
+##### Parameter Optimization on NAB
+
+You can run parameter optimization using your own framework or the framework provided by [htm.core](https://github.com/htm-community/htm.core). As of now, this is only enabled for the htm.core detector, but the same can be done for any detector with low effort (see #792 for details).
+**Requirements**
++ Docker Desktop
++ `pip install docker`
+
+**Usage**
+1. Set `use_optimization = True` in the htm.core detector settings.
+2. Build a docker image from the Dockerfile provided in this repo with `docker build -t optimize-htmcore-nab:latest . -f htmcore.Dockerfile`
+3. Then:
+    + Option A: Check `optimize_bayesopt.py` for an example on how to run with Bayesian Optimization. Note: The script requires `pip install bayesian-optimization`.
```

Comment:
ditto, could be dependency of NAB

Reply:
same as above - do we want to add requirements to NAB which are not really requirements for nab itself?